project-icp/bee-pollinator-app
# Add Google Analytics to Beescape App

## Overview

Rename the generic `GOOGLE_ANALYTICS_ACCOUNT` to the more specific `POLLINATION_GOOGLE_ANALYTICS` for the Pollination Mapper app, and introduce a new `BEEKEEPERS_GOOGLE_ANALYTICS` in the same pipeline for the Beescape app.

We use the same `azaveadev` analytics key for development, and the specified production key for production.

Connects #502

### Demo

![image](https://user-images.githubusercontent.com/1430060/57938910-2b198b80-7897-11e9-9edd-ace90c521898.png)

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Check out this branch and `beekeepers bundle`
* Go to [:8000/?beekeepers](http://localhost:8000/?beekeepers)
* Open dev tools and inspect the HTML
  - [x] Ensure you see Google Analytics properly loading in the app
* Go to [:8000/](http://localhost:8000/)
* Open dev tools and inspect the HTML
  - [x] Ensure you see Google Analytics properly loading in the app
* Inspect [`production.py`](https://github.com/project-icp/bee-pollinator-app/pull/508/files#diff-069fcc54bc52ea70b8c14bcee5c38c29R115)
  - [ ] Ensure the production Google Analytics key matches the issue
